src/server/game/Achievements/CriteriaHandler.cpp

```diff
@@ -1715,6 +1715,13 @@ bool CriteriaHandler::AdditionalRequirementsSatisfied(ModifierTreeNode const* tr
             if (referencePlayer->GetRBGPersonalRating() < reqValue)
                 return false;
             break;
+		case CRITERIA_ADDITIONAL_CONDITION_MAP_DIFFICULTY: // 68
```

Comment:
tabs

Reply:
`tabs2spaces` for all the inserted lines.